packages/styles/forms.css

```diff
@@ -239,6 +239,15 @@ textarea.Field--has-error:focus:hover,
   margin-bottom: 0;
 }
 
+.Field.is--flex-column {
```

Comment:
Why are these additional styles necessary? We haven't had to implement this for any of the other description changes for the other form fields. If there's something we missed we should include it for all of the form fields.

Reply:
It is not needed. it was a miss from a wrapping error earlier.